features/mgmt_api/environment.py

```diff
@@ -0,0 +1,68 @@
+import asyncio
+from superdesk.tests.environment import setup_before_all, setup_before_scenario
+from newsroom.auth_server.oauth2 import generate_jwt_token, config_oauth
+from newsroom.mgmt_api.app import get_app as _get_app
+from newsroom.mgmt_api.default_settings import CORE_APPS, MODULES
+from superdesk.tests import setup as setup_app
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def get_app(*args, **kwargs):
+    return _get_app(*args, **kwargs)
+
+
+class TestClient:
+    client_id = "test"
+
+
+def before_all(context):
+    config = {
+        "BEHAVE": True,
+        "CORE_APPS": CORE_APPS,
+        "INSTALLED_APPS": [],
+        "ELASTICSEARCH_FORCE_REFRESH": True,
+        "MGMT_API_ENABLED": True,
+        "CACHE_TYPE": "null",
+        "MODULES": MODULES,
+    }
+    setup_before_all(context, config, app_factory=get_app)
+
+
+def before_scenario(context, scenario):
+    if "skip" in scenario.tags:
+        scenario.skip("Marked with @skip")
+        return
+
+    try:
+        loop = asyncio.get_running_loop()
+    except RuntimeError:
+        loop = asyncio.new_event_loop()
```

Comment:
Creating a loop here should not be needed. Check how it is currently working with the [NewsAPI tests](https://github.com/superdesk/newsroom-core/blob/async/newsroom/tests/news_api/environment.py#L38).

You will also need to log the exception and re-raise it, so it gets printed to the console